src/spox/opset/ai/onnx/v17.py

```diff
@@ -15867,4 +15867,4 @@ def const(value: npt.ArrayLike, dtype: npt.DTypeLike = None) -> Var:
     "Xor": xor,
 }
 
-__all__ = [fun.__name__ for fun in _CONSTRUCTORS.values()]
+__all__ = [fun.__name__ for fun in _CONSTRUCTORS.values()] + ["const"]
```

Comment:
Is there a reason `_CONSTRUCTORS` only contains ONNX operators and not additions like `const`?

Reply:
Frankly, I can't quite remember why we had the `_CONSTRUCTORS` in the first place. Either way, it seems like a mapping from class names to constructor functions, but `const` does not have a constructor class in the way the other operators do.